taichi/program/state_flow_graph.cpp

```diff
@@ -892,6 +924,12 @@ void StateFlowGraph::topo_sort_nodes() {
           previous_size, nodes_.size());
     }
   }
+  bool changed = false;
```

Comment:
It seems that `changed` is not used?

Reply:
This was for debugging and I forgot to remove. Thanks.